content/en/docs/reference/external-api/custom-metrics.v1beta2.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Kubernetes Custom Metrics (v1beta2)
+content_type: tool-reference
+package: custom.metrics.k8s.io/v1beta2
+auto_generated: true
+---
+<p>Package v1beta2 is the v1beta2 version of the custom_metrics API.</p>
```

Comment:
nit: Is this first sentence in the source code?
Custom Metrics instead of `custom_metrics`?

Reply:
Yup. It is from the source code.